app/javascript/components/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import GreetingComponent from './Greeting';
+
+const App = () => {
+  const greeting = useSelector((state) => state.greeting.greeting);
+
+  return <GreetingComponent greeting={greeting} />;
+};
+
+export default App;
```

Comment:
- [ ] As it is a requirement to create the App component with react-router, I kindly suggest that you please add react-router to your package.json and make use of it here:

<img width="604" alt="image" src="https://github.com/alvp01/hello-rails-react/assets/52934128/9278a999-e0f1-45be-bf0b-29a111376ba5">